src/main/java/net/dv8tion/jda/core/entities/Game.java

```diff
@@ -100,25 +159,33 @@ static Game of(String name, String url)
      *
      * @return True if the provided url is valid for triggering Discord's streaming status
      */
-    static boolean isValidStreamingUrl(String url)
+    public static boolean isValidStreamingUrl(String url)
     {
-        return url != null && url.matches("^https?:\\/\\/(www\\.)?twitch\\.tv\\/.+");
+        return url != null && url.matches("https?://(www\\.)?twitch\\.tv/.+");
     }
 
     /**
      * The type game being played, differentiating between a game and stream types.
      */
-    enum GameType
+    public enum GameType
     {
         /**
          * The GameType used to represent a normal {@link net.dv8tion.jda.core.entities.Game Game} status.
          */
         DEFAULT(0),
         /**
          * Used to indicate that the {@link net.dv8tion.jda.core.entities.Game Game} is a stream, specifically for
+         * <br>This type is displayed as "Streaming" in the discord client.
+         */
+        STREAMING(1),
+        /**
+         * Used to indicate that the {@link net.dv8tion.jda.core.entities.Game Game} is a stream, specifically for
          * <a href="https://www.twitch.tv">https://www.twitch.tv</a>.
          * <br>This type is displayed as "Streaming" in the discord client.
+         *
+         * @deprecated This might be removed when discord introduces more streaming methods. Use {@link #STREAMING} instead.
          */
+        @Deprecated
         TWITCH(1);
```

Comment:
Could actually be removed as users weren't and still aren't able to use it anyway (other than for == checks). 
Might be worth to keep tho in case Discord actually adds different types of streaming. 
Deprecation message could also be reformulated, as removal is not really dependent on discord adding other methods imo (removal due to no-use in lib, just like GameImpl).

Reply:
I don't want to remove it for now. I want to decide later when discord makes their changes (which I expect to happen)